packages/ignite/src/plugins/default/commands/new.js

```diff
@@ -29,35 +29,51 @@ module.exports = async function (context) {
     return
   }
 
+  // Install the ignite-* packages from the local dev directory.
+  //
+  // This is pretty much always what we while we dev.
+  //
+  // To test what live is like, you can run `ignite new FooTown --live`.
+  //
+  // TODO(steve): Don't forget to remove this when we launch... open to better ways of handling it.
+  const igniteDevPackagePrefix = parameters.options.live || `${__dirname}/../../../../../ignite-`
```

Comment:
me gusta means I like it.  I know the meme is a silly face.   I'm saying I like what you did.